iceoryx_posh/include/iceoryx_posh/internal/roudi/port_manager.hpp

```diff
@@ -78,7 +78,8 @@ class PortManager
 
     popo::InterfacePortData* acquireInterfacePortData(capro::Interfaces interface,
                                                       const ProcessName_t& processName,
-                                                      const NodeName_t& node = "") noexcept;
+                                                      const NodeName_t& node = NodeName_t(cxx::TruncateToCapacity,
+                                                                                          "default")) noexcept;
```

Comment:
why `"default"?`

From my point of view, the initialization adds noise and does uglify the code for a not supported compiler. Could you check if `const NodeName_t& node = {""}` works with GCC 5?